tutorials/foreign-object.html

```diff
@@ -0,0 +1,813 @@
+
+            <h1>Foreign Object</h1>
+
+            <p>
+                In the wonderful world of SVG elements, 
+                <a href="https://developer.mozilla.org/en-US/docs/Web/SVG/Element/foreignObject" target="_blank"><code>foreignObject</code></a> 
+                stands out from the rest as it provides some unique behaviour that other SVG elements do not. SVG, like other XML dialects, is
+                namespaced. That means if you want to include elements from a different XML namespace right in your SVG, you need some
+                mechanism to enable this functionality. That's where <code>foreignObject</code> comes into play.
+            </p>
+
+            <p>
+                How does this relate to creating diagrams with JointJS? While JointJS provides a lot of functionality and customization
+                for users, we as developers are a habitual bunch. If we want to create some basic interactive elements, we often want to
+                reach for technologies we are already familiar with, and in web land, the dominant force in this regard is still good old HTML.
+            </p>
+
+            <p>
+                Historically, recommending <code>foreignObject</code> to users of JointJS was a little bit tricky, the main reason for this was the
+                inherent lack of support for <code>foreignObject</code> in Internet Explorer. Due to the necessity of supporting all our users,
+                including those who were still using IE, there was a hesitancy about integrating support into the library fully. Luckily for us, 
+                the days in which we had to support IE are now at an end, and all other major browsers have full support which can be
+                seen in the following <a href="https://caniuse.com/?search=foreignObject" target="_blank">caniuse</a> reference table.
+            </p>
+
+            <p>
+                Now that we can more confidently recommend the use of <code>foreignObject</code> with JointJS, embedding HTML text in SVG, 
+                creating basic interactive elements like buttons, or working with HTML inputs should now be a more straightforward process. 
+                With this addition, combining the power of JointJS with <code>foreignObject</code> opens up a world of possibilities for 
+                interactivity within your JointJS diagrams.
+            </p>
+
+            <p>
+                In order to get a general idea of how <code>foreignObject</code> can be utilized in SVG, let's start with a simple 
+                example without using JointJS.
+            </p>
+
+            <h3>Using foreignObject</h3>
+
+            <p>
+                In the following example, a <code>foreignObject</code> element is placed within our SVG tag. When using HTML elements within
+                <code>foreignObject</code>, note, that it's mandatory to include the XHTML namespace on the outermost or root element. In 
```

Comment:
i think the sentence should be "note that when..., it's mandatory" - it sounds weird right now